### PR TITLE
refactor: EnumStrings.hpp from codingUtilities to common/format

### PR DESCRIPTION
**Pull request by MelReyCG**
This PR is after the #3247, and aim to relocate `EnumStrings.hpp` at a lower level, in the `common/format` package. The goal is to allow basic components, such as the upcoming logger (in `common/logger`), to depend on it, without relying on a circular header dependency.

### CODE REVIEW

**Comment by MelReyCG** [comment]
@rrsettgast @corbett5 @wrtobin @untereiner, may I ask you a review as you guys are the concerned _codeowners_?

File: inputFiles/singlePhaseFlowFractures/fractureFlow_conforming_2d_vtk_input.xml [hunk 1]
```diff
@@ -1,6 +1,7 @@
 <?xml version="1.0" ?>
 
-<Problem>
+<Problem xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+ xsi:noNamespaceSchemaLocation="../../build-pangea4-gcc12.1-hpcxompi2.17.1-onemkl2023.2.0-release/schema.xsd">
```
Comment (untereiner) [inline inputFiles/singlePhaseFlowFractures/fractureFlow_conforming_2d_vtk_input.xml]: this line should not be commited I guess

**Review by untereiner — APPROVED** [review]

**Comment by paveltomin** [comment]
@rrsettgast ?

**Review by rrsettgast — APPROVED** [review]

**Review by paveltomin — APPROVED** [review]

**Review by CusiniM — APPROVED** [review]
LGTM. 

Thanks.

**Pull request merged**